global.json

```diff
@@ -30,6 +31,23 @@
 					"title": "Allwed vxlan range",
 					"format": "^([0-9]{1,8}?-[0-9]{1,8}?)$"
 				}
+			},
+			"operProperties": {
+				"numNetworks": {
+					"type": "int"
+				},
+				"defaultNetwork": {
+					"type": "string"
+				},
+				"vlansInUse": {
+					"type": "string"
+				},
+				"vxlansInUse": {
+					"type": "string"
+				},
```

Comment:
One option is to make `vlanInUse` and `vxlansInUse` fields array of strings instead of just a string. Not a big deal, either way works. If you are planning to summarize ranges, string works better